AirCasting/SessionViews/SessionHeaderView.swift

```diff
@@ -72,24 +72,35 @@ private extension SessionHeaderView {
             //  |_________|     |-------------------|
             // so the idea at leat for now is this below
             #warning("Fix - Handle session.deviceType (for now it is always nill)")
-            switch session.type?.description {
+            sensorType
+                .font(Font.moderate(size: 13, weight: .regular))
+        }
+        .foregroundColor(.darkBlue)
+    }
+    
+    var sensorType: some View {
+        switch session.type.description {
             case "Fixed":
-                Text("\(session.type!.description) : AirBeam3")
-                    .font(Font.moderate(size: 13, weight: .regular))
+                return airBeamText
             case "Mobile":
-                if session.allStreams!.count > 1 {
-                    Text("\(session.type!.description): AirBeam3")
-                        .font(Font.moderate(size: 13, weight: .regular))
+                if session.allStreams!.count == 1 {
```

Comment:
I'm not a fan of this - we can have a session with one stream which isn't mic session - we might have session with removed 4/5 streams created on an android device. So it's not edgy edge case :D 
What would you say about creating sth like this in SessionEntity extension: 
`var isMic: Bool { deviceType =  .MIC }` ? :D 
Might be useful in the future, too :D

Reply:
PS we already have extension with isMobile, isFixed, etc, please add it there ;)